src/components/CommentList.jsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import { useEffect, useState } from "react";
+
+import { getCommentsByID } from "../../utils/api";
+import CommentCard from "./CommentCard";
+
+export default function Comments(props) {
+  const [comments, setComments] = useState([]);
+
+  const { article_id } = props;
+
+  useEffect(() => {
+    getCommentsByID(article_id).then((commentsData) => {
+      setComments(commentsData);
+    });
+  }, []);
```

Comment:
Good work, however please consider adding an `isLoading` here also.

The api call to `getCommentsByID` could still take time to load or fail, so be sure to handle that. Make sure the user can see that comments are loading. Connections will not always be lightning fast, sometimes folks in other countries have quite slow internet so having an isLoading on comments will all support and enhance the user experience.